src/main/java/core/basesyntax/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistrationException extends Exception{
```

Comment:
The class declaration should follow the Java naming conventions and code style practices. The opening brace '{' should be at the end of the same line as the class declaration.